viewer/Viewer.cpp

```diff
@@ -173,6 +252,33 @@ namespace Enki
 		doDumpFrames = doDump;
 	}
 	
+	void ViewerWidget::toogleTrackball()
+	{
+		trackballView = !trackballView;
+		camera.radius = 20;
+	}
+
+	void ViewerWidget::addErrorMessage(const QString& msg, double persistance)
+	{
+		for (std::list<ViewerErrorMessage>::iterator it = messageList.begin(); it!=messageList.end(); it++)
+		{
+			if (it->first == msg)
+			{
+				it->second = persistance;
+				return;
+			}
+		}
+		messageList.push_back(ViewerErrorMessage(msg,persistance));
+	}
+
+	void ViewerWidget::showHelp()
+	{
+		QString str(controlHelp);
+		QStringList slist = str.split("\n");
+		for (int i=0; i<slist.size(); i++)
+			addErrorMessage(slist[i]);
```

Comment:
I need to push sentences separately.
But maybe the function name is misleading.